Assets/Tests/InputSystem/Editor/EditModeTests.cs

```diff
@@ -52,12 +48,12 @@ public void Reset()
     [TestCase(InputSettings.UpdateMode.ProcessEventsManually, InputUpdateType.Manual)]
     [TestCase(InputSettings.UpdateMode.ProcessEventsInDynamicUpdate, InputUpdateType.Dynamic)]
     [TestCase(InputSettings.UpdateMode.ProcessEventsInFixedUpdate, InputUpdateType.Fixed)]
-    #if !ENABLE_XR_COMBINED_DEFINE
-    [Ignore("Must be on an XR-supported platform to run this test.")]
-    #endif
+#if !UNITY_EDITOR
```

Comment:
I don't think this is needed. This is an 'edit mode' test, in an assembly that's only loaded on the editor platform. I know you passed this by me, I forgot about the asmdef/editor test

https://github.com/Unity-Technologies/InputSystem/blob/develop/Assets/Tests/InputSystem/Editor/Unity.InputSystem.Tests.Editor.asmdef